### PR TITLE
obs-studio-plugins.obs-hyperion: fix Function called without required…

### PR DESCRIPTION
**Pull request by Artturin**
… argument "qtbase"

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
   buildInputs = [ obs-studio libGL qtbase ];
 
+  dontWrapQtApps = true;
+
+  cmakeFlags = [
+    "-DOBS_SOURCE=${obs-studio.src}"
+  ];
+
   meta = with lib; {
     description = "OBS Studio plugin to connect to a Hyperion.ng server";
```
Comment (SuperSandro2000) [inline pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix:25]:
```suggestion
    description = "OBS Studio plugin to connect to a Hyperion.ng server";
    homepage = "https://github.com/hyperion-project/hyperion-obs-plugin";
```

File: pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
   version = "1.0.1";
 
-  src = fetchFromGitLab {
+  src = fetchFromGitHub {
     owner = "hyperion-project";
     repo = "hyperion-obs-plugin";
-    rev = "v${version}";
-    sha256 = "sha256-Si+TGYWpNPtUUFT+M571lCYslPyeYX92MdYV2EGgcyQ=";
+    rev = version;
+    sha256 = "sha256-pfWfJWuIoa+74u5J76/GE+OuHkksbwOAPfsR9OGX3L4=";
```
Comment (SuperSandro2000) [inline pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix:12]: Requires submodule see https://github.com/hyperion-project/hyperion-obs-plugin/issues/7

Preferred way would be to symlink our flatbuffers.

Comment (Artturin) [reply pkgs/applications/video/obs-studio/plugins/obs-hyperion/default.nix:12]: Thanks

**Comment by Algram** [comment]
Thanks for picking up my broken Draft :heart: 

I think this still needs the flatbuffers version bumped to work correctly:https://github.com/NixOS/nixpkgs/pull/188951

**Pull request merged**